modules/search/instant-search/README.md

```diff
@@ -1,20 +1,20 @@
 # Jetpack Instant Search
 
-The **Jetpack Instant Search"** module currently consists of a JavaScript app built on **Preact**. In its current form, it enhances WordPress Search pages by presenting search results while the user is still typing.
+The **Jetpack Instant Search"** module consists of a JavaScript app built on **Preact**.
 
-Eventually, we expect this module to enhance all front-end views that enable the user to quickly jump into a full-page search experience.
+It enables the user to quickly jump into a full-page search experience with filtering, sorting and instant results.
 
 ## Development
 
 ### High-level overview of the development flow
 
 1. Use the [Jetpack Docker environment](https://github.com/Automattic/jetpack/tree/master/docker#readme).
 2. Start a new branch.
-3. Modify/Improve the code in the instant-search directories. New libs get added to the common package.json.
+3. Modify/improve the code in the instant-search directories. New packages should be added to the common package.json.
 4. Run `yarn build-search [--watch]` to compile your changes.
-5. Now test your changes on the front end of your test site.
+5. Test your changes on the front-end of your test site.
 6. Open a PR, and a WordPress.com diff will be automatically generated with your changes.
-7. Test the WordPress.com diff
+7. Test the WordPress.com diff.
 8. Once the code works well in both environments and has been approved by a Jetpack crew member, you can merge your branch!
 
 ### Basic Build Process
```

Comment:
Once #15748 lands, we could just go with `yarn watch` rather than the two `build` steps.